### PR TITLE
Convert the repo into a install-able module

### LINKED ISSUE #22: Convert this repo into a module
It would be nice if this repo was maintained as a commonjs module like the [lancer-data](https://github.com/massif-press/lancer-data) repo is. I have made the necessary changes on [my fork](https://github.com/OmnesPotens/long-rim-data), but I also added some github workflow files which may not be desired. 

I also published some packages for the various official player LCPs, I hope thats ok. If not, I can unpublish them!

### PR DESCRIPTION
**Pull request by OmnesPotens**
PR intended to close #22.

Notes: 
- I have published packages for the core player LCPs, if that is not ok or you'd like me to unpublish them I can and will
- These changes include github workflow files that may not be desired

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "homepage": "https://github.com/OmnesPotens/long-rim-data",
+  "scripts": {
+    "build-gpr": "node ./.github/gpr-publishing.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "lancer",
+    "nrfaw",
+    "no room for a wallflower",
```
Comment (msprijatelj) [inline package.json:26]: Heads up: These Wallflower keywords may be inappropriate for The Long Rim.

**Review by msprijatelj — COMMENTED** [review]
I want to better understand the benefits of making this a commonjs package. One minor benefit of the current repo structure is that it can easily be downloaded as a zip and renamed to an LCP for installation into CompCon. That could still be accomplished with GitHub Releases, but it’s something to think about.

**Comment by git-ival** [comment]
Thanks for taking the time to comment + review! 

The idea was that any interested developers could use this as a dependency in potential projects. I can see the lancer-data module is already being used to facilitate other less mechanical game aids. 

For example: [Kuenaimaku's lancer-briefings](https://github.com/Kuenaimaku/lancer-briefings/blob/feature/persons-register/package.json) uses the `lancer-data` module. I also use `lancer-data` and the other LCP data modules that I packaged in my fork of [`lancer-briefings` here](https://github.com/OmnesPotens/lancer-briefings/blob/feature/persons-register/package.json).

**Comment by OmnesPotens** [comment]
Thanks for taking the time to comment + review! 

The idea was that any interested developers could use this as a dependency in potential projects. I can see the lancer-data module is already being used to facilitate other less mechanical game aids. I think making the data more accessible from a developer standpoint is low-effort, low or no-maintenance and I don't think it would reduce ease of packaging as LCP zips.

For example: [Kuenaimaku's lancer-briefings](https://github.com/Kuenaimaku/lancer-briefings/blob/feature/persons-register/package.json) uses the `lancer-data` module. I also use `lancer-data` and the other LCP data modules that I packaged in my fork of [`lancer-briefings` here](https://github.com/OmnesPotens/lancer-briefings/blob/feature/persons-register/package.json).

File: package.json (hunk 1 above)
Comment (OmnesPotens) [reply package.json]: Fixed

**Comment by jarena3** [comment]
turns out shortly was this afternoon! Thanks @OmnesPotens :)

**Pull request closed without merging**